hack/upgrade-e2e-test.sh

```diff
@@ -20,9 +20,11 @@
 go run "$(dirname $0)/e2e.go" -v -build -up -version="v0.14.0" -test -check_version_skew=false
 if [ $? -eq 0 ]; then
 	echo "Tests on initial version succeeded. Proceeding with push and second set of tests."
-	go run "$(dirname $0)/e2e.go" -v -push -version="" -test -check_version_skew=false
+	go run "$(dirname $0)/e2e.go" -v --test --test_args='--ginkgo.focus=Skipped.*Cluster.*upgrade' -check_version_skew=false
```

Comment:
I think it makes sense to use this file for node upgrades / tests as well, so it's probably not worth renaming the file, but for the interim, please make it painfully clear this only does master upgrades.